team-01/src/main/java/ubc/cosc322/AmazonsLocalBoard.java

```diff
@@ -59,4 +60,11 @@ private int getIndex(ArrayList<Integer> position) {
 		int col = position.get(1);
 		return TOTAL_LENGTH - row * ROW_LENGTH + col;
 	}
+
+	public AmazonsLocalBoard getDeepCopy() {
```

Comment:
I'd probably just call this `copy()`, for conciseness. `deepCopy()`, if you really want to be explicit, though it's not necessary in my opinion.

Reply:
Sure, I can rename it, I'm fairly neutral, although I like that `getDeepCopy` immediately lets you know what you're getting: less ambiguous.